core/templates/pages/admin-page/platform-parameters-tab/admin-platform-parameters-tab.component.ts

```diff
@@ -143,12 +151,47 @@ export class AdminPlatformParametersTabComponent implements OnInit {
     return {type: dataType};
   }
 
-  addNewRuleToTop(param: PlatformParameter): void {
-    param.rules.unshift(cloneDeep(this.getdefaultNewRule(param)));
+  getReadonlyFilterValues(rule: PlatformParameterRule): string {
+    let resultantString: string = '';
+    let filters: PlatformParameterFilter[] = rule.filters;
+    for (let filterIdx = 0; filterIdx < filters.length; filterIdx++) {
+      let filterName: string = (
+        this.filterTypeToContext[filters[filterIdx].type].displayName);
+      if (filters[filterIdx].conditions.length === 0) {
+        resultantString += filterName + ' in ' + '[ ]';
+      } else {
+        let conditions: string = '';
+        for (let idx = 0; idx < filters[filterIdx].conditions.length; idx++) {
+          if (idx === filters[filterIdx].conditions.length - 1) {
+            conditions += filters[filterIdx].conditions[idx][1];
+          } else {
+            conditions += filters[filterIdx].conditions[idx][1] + ', ';
+          }
+        }
+        if (filterIdx === filters.length - 1) {
+          resultantString += (
+            filterName + ' in ' + '[' + conditions + ']');
```

Comment:
Use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) everywhere you are concatenating multiple strings. They are more readable.